build/CMakeLists.txt

```diff
@@ -3,13 +3,22 @@
 # @author Pere Mato, CERN
 ############################################################################
 
-file(GLOB rmkdepend_srcs ${CMAKE_CURRENT_SOURCE_DIR}/rmkdepend/*.cxx ${CMAKE_CURRENT_SOURCE_DIR}/rmkdepend/*.c )
-set_source_files_properties(${rmkdepend_srcs} PROPERTIES 
-                                              COMPILE_DEFINITIONS "INCLUDEDIR=\"/usr/include\";OBJSUFFIX=\".o\";USE_CHMOD")
-
-ROOT_EXECUTABLE(rmkdepend ${rmkdepend_srcs})
-
 if(WIN32)
-  ROOT_EXECUTABLE(bindexplib  ${CMAKE_CURRENT_SOURCE_DIR}/win/bindexplib/*.cxx)
+  ROOT_EXECUTABLE(bindexplib win/bindexplib/bindexplib.cxx)
 endif()
 
+ROOT_EXECUTABLE(rmkdepend
+  rmkdepend/cppsetup.c
+  rmkdepend/ifparser.c
+  rmkdepend/include.c
+  rmkdepend/main.c
+  rmkdepend/mainroot.cxx
+  rmkdepend/parse.c
+  rmkdepend/pr.c
+)
+
+target_compile_definitions(rmkdepend PRIVATE
+  INCLUDEDIR="${CMAKE_INSTALL_INCLUDEDIR}"
```

Comment:
I think that this is only defined so that `rmkdepend` compiles. It doesn't really matter. It was hard-coded to `/usr/include`. Maybe I will just replace `INCLUDEDIR` with `"/usr/include"` directly in the code and define the other macros so we can remove the `target_compile_definitions()` altogether.

Reply:
Done.

Reply:
Could you verify by looking at the code that it's just an arbitrary dir? Can we keep it at `/usr/local`?